docker/app/Dockerfile

```
@@ -28,4 +28,4 @@ CMD ["java", "-cp", "app:app/lib/*", "io.tolgee.Application"]
 
 # Health check to ensure the app is up and running
 HEALTHCHECK --interval=10s --timeout=3s --retries=20 \
-    CMD curl -f http://127.0.0.1:$HEALTHCHECK_PORT/actuator/health || exit 1
+    CMD wget -q -O - http://127.0.0.1:$HEALTHCHECK_PORT/actuator/health || exit 1
```

Thank you for your contribution, it looks working :)

Minor comment: `wget` can be called with the `--spider` parameter not to download the content, just check the HTTP code.

Also the `|| exit 1` is redundant, since it is the default `wget` functionality.

```suggestion
    CMD wget --spider -q http://127.0.0.1:$HEALTHCHECK_PORT/actuator/health
```

---

It is not redundant; wget doesn't always exit with exit code 1 which is the expected code for unhealthy containers, not just non-zero. While it works, the Dockerfile reference explicitly expects an exit code 1, reserves the exit code 2, and makes no mention of other exit codes making them effectively undefined behavior. 

- https://docs.docker.com/reference/dockerfile/#healthcheck
- https://man.archlinux.org/man/wget.1.en#EXIT_STATUS

---

@StaNov Can you please finalize this so we can merge it? 🙏

---

I have to create a new PR because I cannot work on this branch.

@vexdev FYI, if you were able to finish it in a matter of hours, I'd leave it to you.

@cyyynthia, thanks for the explanation, I didn't know this healthcheck contract.

---

FYI, I created a new PR #2694. I credited @vexdev in the description.

---

No worries I'm not after credits only helpful contributions when I can :)